test/suite/test_turtle.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
I think your test only verifies for the empty database. We should also verify the case that we have done a metadata checkpoint.

Reply:
I added some logic to create a table, perform some writes and then do a checkpoint. This way we cover both the empty and non-empty database cases.  Is what I have sufficient?